### ISSUE TITLE
Invalid screen

### ISSUE THREAD

**Issue by anonpan** [dice4321/Gunkolle#19]
When I run Gunkolle.ahk it detects Nox and it says "Window found" but then it says "Invalid Screen, Retrying (1)" and I can't get the "Ready". I'm on the game's home screen, should I be somewhere else? The game is running at 1280 x 720 and speed mode.

**Comment by dice4321** [comment]
Are you running an AMD GPU? If so, this script wont work because of how NOX renders images differently with an AMD GPU.

**Comment by anonpan** [comment]
nope, Nvidia.

**Comment by dice4321** [comment]
home screen with the the T-doll secretary right? Maybe try restoring window size? 
![alt text](https://i.imgur.com/tC8rkPt.png)

**Comment by anonpan** [comment]
I tried that too. I even tried changing my secretary because I thought maybe she was covering a specific area that shouldn't be covered but it didn't work either 🤷

**Comment by anonpan** [comment]
I managed to fix that problem but I have a new one now. I get the "Ready" but then when I click the "Start" button, a pop up saying "Window "NoxPlayer" was not found" appears.

**Comment by anonpan** [comment]
The last update fixed that issue but now I'm having another problem. It detects Nox and everything, I get the 'Ready', the macro starts running, goes to formation, changes the leader (not always, sometimes it gets stuck after filtering the dolls), goes to the battle screen, selects 4-3e, deploys the echelons, resupplies and everything but then it selects the first two enemies, scrolls the map and it stops there. It doesn't select the other two enemies.

![image](https://user-images.githubusercontent.com/42301429/45450725-5a10a980-b6d9-11e8-9d58-232701a6dfff.png)

This is where it gets stuck. Maybe it's because of the way I set the drag of the map? I tried changing it a couple of times but it still doesn't work.

![image](https://user-images.githubusercontent.com/42301429/45450892-c5f31200-b6d9-11e8-9f58-7721aaac300a.png)
